src/style.css

```diff
@@ -175,8 +175,8 @@ tr:nth-child(even) {
   margin-bottom: 3rem;
 }
 
-.spinner {
-  border: 4px solid #c1bcca;
+.RotatorReload {
```

Comment:
- [ ] for better UI design I would suggest adding a limit to your list height and enabling overflow to it, to avoid scrolling outside the main container and to make your design look better: 

```CSS
.list-players {
  border: 0.125rem solid #333;
  border-radius: 0.25rem;
  max-width: 32rem;
  margin: 0 auto;
  margin-bottom: 4rem;
  /* Add this Example or modify it as you like */
   /* Example Start */
  max-height: 40vh;
  overflow-x: hidden;
  overflow-y: scroll;
   /* Example End */
}
```

**Result:**

Before           | After
:-------------------------:|:-------------------------:
![afffff](https://user-images.githubusercontent.com/89282221/195576801-11d01fbb-650b-4954-a2d3-64ff4065c25c.gif) | ![bef](https://user-images.githubusercontent.com/89282221/195576536-90e30457-7d76-488e-ad78-14a3a3f7a021.gif)


**_kindly fix it to make your website match the wireframe provided in the requirements_**

Reply:
Hello @ichala 👋 
Thanks for this suggestion. I've implemented it, and the app looks good now 😃